docs/developers-guide/driver-changelog.md

```diff
@@ -50,9 +50,20 @@ title: Driver interface changelog
    - `metabase.query-processor.compile/compile-and-splice-parameters` has been removed; replace usages with
      `metabase.query-processor.compile/compile-with-inline-parameters`.
 
-- The three-arity of `metabase.driver.sql.query-processor/format-honeysql` (which had an additional parameter for
-  Honey SQL version) has been removed; replace all usages with the two-arity version. Honey SQL 2 has been the only
-  supported version since Metabase 0.49.0.
+  - The three-arity of `metabase.driver.sql.query-processor/format-honeysql` (which had an additional parameter for
+    Honey SQL version) has been removed; replace all usages with the two-arity version. Honey SQL 2 has been the only
+    supported version since Metabase 0.49.0.
+
+## Metabase 0.50.13
```

Comment:
this is speculative. I might need to update this depending on what version it will actually go out in